script-desc.js

```diff
@@ -0,0 +1,109 @@
+const img5 = document.createElement('img');
+const detailDesc = document.createElement('section');
+const projDesc = document.createElement('p');
+const btnContainer = document.createElement('div');
+const btnLive = document.createElement('button');
+const btnSource = document.createElement('button');
+const btnDirection = document.createElement('div');
+const btnPrev = document.createElement('button');
+const btnNext = document.createElement('button');
+
+headerImg.src = projects[0].headerImg;
+headerImg.alt = projects[0]['headerImg-alt'];
+title.textContent = projects[0].name;
+[workDescItem1.textContent,
+  workDescItem2.textContent,
+  workDescItem3.textContent] = projects[0].technologies;
+img1.src = projects[0].image;
+img1.alt = projects[0]['workImg-alt'];
+img2.src = projects[0].image;
+img2.alt = projects[0]['workImg-alt'];
+img3.src = projects[0].image;
+img3.alt = projects[0]['workImg-alt'];
+img4.src = projects[0].image;
+img4.alt = projects[0]['workImg-alt'];
+img5.src = projects[0].image;
+img5.alt = projects[0]['workImg-alt'];
+projDesc.textContent = projects[0].description;
+btnLive.textContent = 'See live';
+btnSource.textContent = 'See source';
+btnPrev.textContent = 'Previous project';
+btnNext.textContent = 'Next project';
+
+container.appendChild(smallContainer);
+smallContainer.append(headerBtn, header, gridContainer, detailDesc);
+headerBtn.appendChild(headerImg);
+header.append(title, workDescList);
+workDescList.append(workDescItem1, workDescItem2, workDescItem3);
+gridContainer.appendChild(grid);
+grid.append(fig1, fig2, fig3, fig4, fig5);
+fig1.appendChild(img1);
+fig2.appendChild(img2);
+fig3.appendChild(img3);
+fig4.appendChild(img4);
+fig5.appendChild(img5);
+detailDesc.append(projDesc, btnContainer, btnDirection);
+btnContainer.append(btnLive, btnSource);
+btnDirection.append(btnPrev, btnNext);
+
+smallContainer.className = 'small-container';
+headerBtn.className = 'headerBtn';
+headerBtn.setAttribute('id', 'close-popup');
+header.className = 'header';
+title.className = 'title';
+workDescList.className = 'workDesc-list';
+workDescItem1.className = 'workDesc-item';
+workDescItem2.className = 'workDesc-item';
+workDescItem3.className = 'workDesc-item';
+gridContainer.className = 'grid-container';
+grid.className = 'grid';
+fig1.className = 'fig1';
+fig2.className = 'fig2';
+fig3.className = 'fig3';
+fig4.className = 'fig4';
+fig5.className = 'fig5';
+img1.className = 'work-img';
+img2.className = 'work-img';
+img3.className = 'work-img';
+img4.className = 'work-img';
+img5.className = 'work-img';
+detailDesc.className = 'detail-desc';
+btnContainer.className = 'btn-cont';
+btnLive.className = 'btn-live';
+btnSource.className = 'btn-source';
+btnDirection.className = 'direction-btn invisible';
+
+// close popup script
+
+const closePopup = document.getElementById('close-popup');
+
+function closePopupFunc() {
+  window.close();
+}
+
+closePopup.addEventListener('click', closePopupFunc);
```

Comment:
- [ ] In order to have DRY code kindly consider not having different js files for each pop-up by using a single function to map into your projects arrays and create each pop-up as their structures are the same use a single function to iterate over your array and that will make your codebase professional  check below all the number of the js files you have which are doing the same work and fix all of them into one 
<img width="557" alt="Screen Shot 2023-06-28 at 09 22 48" src="https://github.com/binyamolango/Personal-Portfolio/assets/60146030/8708d1f6-a145-4ead-8d33-40cc2512845c">



while doing so consider doing the same on your projects html files as well to avoid repeating yourselves